### PR TITLE
Update Slack bot tutorial to use internal integrations

### PR DESCRIPTION
**Pull request by tswast**

### CODE REVIEW

File: tutorials/run-botkit-on-google-container-engine/index.md [hunk 1]
```diff
@@ -69,54 +69,54 @@ The code for this tutorial is [hosted on GitHub][repo].
...
-    typical user tokens.
+1.  Create a [new Slack app](https://api.slack.com/apps).
+    1.  Give the app a name, such as "Kittenbot".
+    1.  Choose the Slack team where you want it installed.
+1.  Add a new bot user to the app.
+    1.  Select **Bot users** under the features heading on the left-hand side
+        navigation of the app configuration page.
+    1.  Click the **Add a bot user** button.
+    1.  Give it a nice username, like @kittenbot.
+    1.  This tutorial uses the RTM API, so do not always show the bot as
```
Comment (JimTravis) [inline tutorials/run-botkit-on-google-container-engine/index.md]: Replace "do" with "does" ?

Comment (tswast) [inline tutorials/run-botkit-on-google-container-engine/index.md]: It's actually an option in the dialog. I've updated the sentence to clarify.

**Comment by jmdobry** [comment]
So the images deleted here can also be removed from the Cloud Storage bucket?

**Comment by tswast** [comment]
Yeah, those images for the custom integrations are no longer relevant. (At least the won't be after this is published.)

**Pull request merged**